### DISCUSSION TITLE
Can edgy.nvim be used to create and manage multiple window layouts?

### DISCUSSION THREAD

**Discussion by MoltenMonster** [folke/edgy.nvim#92] [Q&A]
The readme isn't very clear.

**Comment by folke** [comment]
no

**Reply by MoltenMonster** [reply]
It says right in the description "Easily create and manage predefined window layouts". 

That's what I want. For instance, when I open a markdown doc I want Neotree and Outline.nvim open too in their own windows, and if I open a code file (perhaps with predefined file extensions), it opens Trouble in its own window. I wouldn't want Trouble open for Markdown. 

Can this be achieve with Edgy?

**Reply by MoltenMonster** [reply]
@folke

**Reply by HarshalRathore** [reply]
@Oneechan69 i thought the same reading the readme i thought it can be used to manage the window layouts like it has some efficient predefined window layouts which we can use.

**Comment by peterwilli** [comment]
Yeah this is kind of a let-down. I travel a lot and it would've been nice to be able to switch different layouts depending if I am connected to a large monitor or not. This is really something I would have wanted! Guess I'll keep looking...

Edit 1: I checked out different projects, namely nvim-ide, but it doesn't fit my profile either. It is mainly focussed on creating its own ecosystem of components rather than using the (vast) neovim generic ecosystem. This brought me back here to Edgy. I propose the mainatiner to update the readme to change "layouts" to "layout", so it is more clear what this project is about, it is also better to be more fair upfront so the people that land on your project stick around!

In the meantime, I'm trying to make a as close as possible layout that I wished to have with this plugin, which is still great! Once I'm satisfied with a single layout, I'll check if I can alter the plugin to add multiple layouts...

Edit 2:

I managed to get a really nice layout going! It's based on the lazyvim config, I only made minor changes. I feel this layout will work well on smaller monitros, but for my wide monitor I wish to move the 2 bottom panes to the side, which I will do now. I'll just comment the old layout out for now, but I'll check later if I can change Edgy to toggle seamlessly between layouts.
![Screenshot of my current Edgy layout](https://github.com/user-attachments/assets/ab0ca8bf-f96f-4ed5-a1f6-9bafeda4620d)
